README.md

```diff
@@ -1,189 +1,67 @@
-2. Click **Create a config store**.
-3. Name the config store *exactly* `Fingerprint` (the store name is hard-coded in the service implementation) and click **Add**. 
-4. Click **Link to services** and select `fingerprint-pro-fastly-proxy-integration`.
-5. Click **Next**, select the current (draft) version of your service, and click **Link only**.
-6. Click **Finish**.
-7. Find your new config store and click **Key-value pairs** to add the following values:
-   - set `AGENT_SCRIPT_DOWNLOAD_PATH` to your chosen agent download path. It should be something random to avoid ad blockers, for example, `463n7-d0wnl04d`.
-   - set `GET_RESULT_PATH` to your chosen identification result path. It should be something random to avoid ad blockers, for example, `1d3n71f1c4710n-r35ul7`.
-   - set `PROXY_SECRET` to the value of your Fingerprint proxy secret you created in Step 1.
-   - set `OPEN_CLIENT_RESPONSE_ENABLED` to `true` if open client response is enabled in your subscription; otherwise, set it to `false`.
+   | Key                          | Example Value        | Description                                                                                 |
+   |------------------------------|----------------------|---------------------------------------------------------------------------------------------|
+   | PROXY_SECRET                 | 6XI9CLf3C9oHSB12TTaI | Fingerprint proxy secret generated in Step 1                                                |
+   | OPEN_CLIENT_RESPONSE_ENABLED | false                | open client response is enabled in your subscription, true for enabled, false for otherwise |
+   | AGENT_SCRIPT_DOWNLOAD_PATH   | z5kms2               | Random path segment for downloading the JS agent                                            |
+   | GET_RESULT_PATH              | nocmjw               | Random path segment Fingerprint identification requests                                     |
 
-### 8. Create a domain for the service
+3. Go to [Releases](https://github.com/fingerprintjs/fingerprint-pro-fastly-compute-proxy-integration/releases) to download the latest `fingerprint-proxy-integration.tar.gz` package file.
+4. Upload package to your Fastly Compute Service's **Package**.
+5. Configure the Fingerprint [JS Agent](https://dev.fingerprint.com/docs/js-agent) on your website using the paths defined in Step 2.
+    ```javascript
+   import * as FingerprintJS from '@fingerprintjs/fingerprintjs-pro'
 
-We recommend using a subdomain of the website you want to use Fingerprint on. Do not use `fingerprint`, `fpjs`, and other fingerprint-related terms in the subdomain to avoid ad blockers. Use something random or generic, for example: `metrics.yourwebsite.com`.
+   const fpPromise = FingerprintJS.load({
+     apiKey: 'PUBLIC_API_KEY',
+     scriptUrlPattern: [
+       'https://yourwebsite.com/INTEGRATION-PATH/AGENT_SCRIPT_DOWNLOAD_PATH?apiKey=<apiKey>&version=<version>&loaderVersion=<loaderVersion>',
```

Comment:
This is the first mention of `INTEGRATION-PATH` path on this page, are you sure it's supposed to be here?

Reply:
Removed the mention of integration path and added subdomain

Reply:
If everything is okay, can you resolve this comment and others if any other unresolved comments are left?